exercises/rxjs-observable/README.md

```diff
@@ -0,0 +1,34 @@
+# RxJS Exercises
+
+## Exercise 1
+
+Create an RxJS `Observable` to display 2 different integers using `setInterval` and `next`.
+
+## Exercise 2
+
+Using the `map` operator create an RxJS `Observable` that will take an array of numbers and multiply each item by 2. 
+
+## Exercise 3
+
+Using the `filter` operaor create an RxJS `Observable` that will take an array of numbers from 1 to 10 and filters it to return any number greater than 5. 
+
+## Exercise 4
+
+Create an RxJS `Observable` that will output a number and then throw an error. 
+
+## Exercise 5
+
+Create an Observable with the string `hello` and another one with the word `world`. 
+Merge the second observable's and print out `hello world`.
+
+## Exercise 6
```

Comment:
Oh, I actually meant this comment for Exercise 6, but I like the changes you made to #5 as well

I find 6 unclear